README.md

```diff
@@ -42,10 +42,11 @@ You should also specify settings that will be shared across all the plugin rules
       "version": "15.0", // React version, default to the latest React stable release
       "flowVersion": "0.53" // Flow version
     },
-    "propWrapperFunctions": [ "forbidExtraProps" ] // The names of any functions used to wrap the
-                                                   // propTypes object, e.g. `forbidExtraProps`.
-                                                   // If this isn't set, any propTypes wrapped in
-                                                   // a function will be skipped.
+    "propWrapperFunctions": [                      // The names of any functions used to wrap the
+        "forbidExtraProps",                        // propTypesObject, e.g. `forbidExtraProps`
+        {"property": "freeze", "object": "Object"} // If this isn't set, any propTypes wrapped in
```

Comment:
This comment is meant to be all one line - let’s move all the comment bits to a single comment inside the array?